src/odemis/driver/smarpod.py

```diff
@@ -870,3 +1117,568 @@ def Smarpod_GetDLLVersion(self, p_major, p_minor, p_update):
         minor.value = 2
         update = _deref(p_update, c_uint)
         update.value = 3
+
+
+class SmarAct_MC(model.Actuator):
+
+    def __init__(self, name, role, locator, ref_on_init=False, linear_speed=0.01,
+                 rotary_speed=1, axes=None, **kwargs):
+        """
+        A driver for a SmarAct SA_MC Actuator.
```

Comment:
SA_MC -> MC. Please explain what the actuator is. That's the "Motion Controller" which formally was called "TriGlide", and is a 5DoF custom-designed stage.